Artifacts/windows-seleniumgrid-node/Artifactfile.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "Selenium-grid node",
+    "description": "Use this artifact to setup your selenium-grid node",
+    "publisher": "Microsoft",
+    "tags": [
+        "PowerShell",
+        "Selenium"
+    ],
+    "iconUri": "https://storagehsm6ciz.blob.core.windows.net/mycustomscripts/selenium-grid-logo.png",

```

Comment:
Ditto the comment from above.  Also, we prefer not to have logos checked into our tree as it might infringe on others IP rights